MediaAdmin/Security/Authorization/Voter/MediaFolderGroupRoleVoter.php

```diff
@@ -85,7 +85,7 @@ public function vote(TokenInterface $token, $object, array $attributes)
             if (!$group->getSite() instanceof ReadSiteInterface) {
                 continue;
             }
-            if (!$object->hasSite($group->getSite()->getSiteId())) {
+            if ($object->getSiteId() != $group->getSite()->getSiteId()) {
```

Comment:
you should split ``$group->getSite()->getSiteId()`` into 2 lines, to check that ``$group->getSite()`` is not null before trying to access ``getSiteId()``
It will avoid some 500 errors